__tests__/app.test.js

```diff
@@ -80,6 +80,24 @@ describe("/api/articles/:article_id", () => {
             expect(body.msg).toBe("Bad Request")
         });
     });
+    it("GET 200: a valid article query response includes the correct comment_count if comments for that article exist", () => {
+        return request(app)
+        .get("/api/articles/1")
+        .expect(200)
+        .then(({ body }) => {
+            expect(body.article).toHaveProperty("comment_count", expect.any(Number))
+            expect(body.article.comment_count).toBe(11)
```

Comment:
If we're explicitly testing that the comment count is 11 on line 89, then line 88 will always automatically pass because we must have a property of comment_count which is indeed a number (11)